components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/SQLQueries.java

```diff
@@ -500,6 +500,59 @@ public class SQLQueries {
             "JOIN IDN_OAUTH2_ACCESS_TOKEN_SCOPE ON ACCESS_TOKEN_TABLE.TOKEN_ID = " +
             "IDN_OAUTH2_ACCESS_TOKEN_SCOPE.TOKEN_ID";
 
+    /**
+     * SQL query to retrieve access tokens for a given consumer key and scope.
+     * It selects various attributes of access tokens from the IDN_OAUTH2_ACCESS_TOKEN table.
+     * The query involves joining multiple tables:
+     * - It selects access tokens from a subquery filtering by consumer key ID and token state.
+     * - It joins the resulting access token table with the IDP table based on the IDP ID.
+     * - It left joins the access token scope table to retrieve token scope information.
+     * The query parameters are placeholders for the consumer key, tenant ID, token state, and token scope.
+     */
+    public static final String  GET_ACCESS_TOKENS_FOR_CONSUMER_KEY_AND_SCOPE =
```

Comment:
Format the query in a consistent way with the other queries.

Reply:
Added